jackbox_patcher/assets/l10n/app_de.arb

```diff
@@ -127,10 +127,10 @@
     "connection_to_server_failed": "Verbindung zum Server fehlgeschlagen",
     "connection_to_main_server_failed": "Aufgrund eines Fehlers konnten die verfügbaren Server nicht abgerufen werden.",
     "quit_while_downloading_title": "Ein Download ist aktiv",
-    "quit_while_downloading_description": "Ein Download ist aktiv, bist du sicher, dass du das Programm schließen möchtest?",
+    "quit_while_downloading_description": "Ein Download ist zurzeit aktiv, bist du sicher, dass du die Anwendung schließen möchtest?",
     "quit": "Beenden",
-    "automatic_game_finder_title": "Automatische Spiele-Erkennung",
```

Comment:
Let's keep "Automatische Spiele-Erkennung", since it's detecting more than one game

Reply:
Used the Singular here because, in english it uses the singular aswell. But i'm fine with both tho. In the end it's a preference in wording while using the Singular sounds better imho.